stackslib/src/chainstate/stacks/tests/mod.rs

```diff
@@ -555,13 +555,13 @@ impl TestStacksNode {
         burn_block: &mut TestBurnchainBlock,
         miner: &mut TestMiner,
         stacks_block: &StacksBlock,
-        microblocks: &Vec<StacksMicroblock>,
+        microblocks: &[StacksMicroblock],
```

Comment:
Again just pass a `Vec` here:

```suggestion
        microblocks: Vec<StacksMicroblock>,
```

As a general rule in Rust, if a function clones or makes an owned collection from a reference argument, it's better to pass the argument as an owned type. This allows the caller decide whether to give up ownership or construct the owned type

Reply:
Yeah we have a bunch of spots where we do this. Either creating vectors from arrays or taking a reference then immediately cloning it.